src/space_stage/SpaceStage.cs

```diff
@@ -280,14 +280,15 @@ public void SetupForExistingGameFromAnotherStage(bool spawnPlanet, UnitType spac
         if (spawnPlanet)
             AddPlanet(Transform3D.Identity, true);
 
-        var fleet = AddFleet(new Transform3D(Basis.Identity, new Vector3(6, 0, 0)),
+        var fleet = AddFleet(new Transform3D(Basis.Identity, new Vector3(20, 0, 0)),
             spacecraft, true);
 
         // Focus the camera initially on the ship to make the stage transition smoother
         ZoomOutFromFleet(fleet);
 
+        // TODO: Set launched craft to move from the planet
```

Comment:
Was this intended to not be completed for this?

Reply:
As it's now switching to a separate ship, I felt it was unnecessary. I could revert that if you think it'd look better.

Reply:
I think it looks much better with the unit moving initially. And this code would need to be fixed up and re-enabled when the transition from industrial to space is being worked on in the far future.